StortSpelProjekt/Engine/src/Renderer/HLSL/ForwardPixel.hlsl

```diff
@@ -96,7 +96,7 @@ PS_OUTPUT PS_main(VS_OUT input)
 	float3 ambient = float3(0.004f, 0.004f, 0.004f) * albedo;
 	finalColor += ambient;
 
-	finalColor += emissive.rgb;
+	finalColor += (emissive.rgb * 1000);
 
```

Comment:
I would like you to quickly write a comment explaining why we * 1000 and quickly explains the limitations.